src/psyclone/psyir/nodes/node.py

```diff
@@ -57,6 +57,7 @@
 # We use the termcolor module (if available) to enable us to produce
 # coloured, textual representations of Invoke schedules. If it's not
 # available then we don't use colour.
+# no cover: start
 try:
     # pylint disable=import-outside-toplevel
     from termcolor import colored
```

Comment:
Can't we monkeypatch the packages that appear to be available from `termcolor`? I had a google and found https://stackoverflow.com/questions/8658043/how-to-mock-an-import